### ISSUE TITLE
ne_10m_ports  | missing ne_id column

### ISSUE THREAD

**Issue by ImreSamu** [nvkelso/natural-earth-vector#462]
As I see 
-  in the "ne_10m_ports" table - the new `ne_id` column is still missing.  

( my refactored code - now -  expecting `ne_id` column  "everywhere"  ... and after I can create some basic wikidataid suggestions   )

![image](https://user-images.githubusercontent.com/2217320/103436551-2268fb00-4c1d-11eb-9891-af93d0b79011.png)

but the matching is very low ...

```
select * from newd.ne_wd_match_ports_match_agg_sum_pct;
+-----------------------------------------------------+------+----------+
|                 _matching_category                  |  n   |   pct    |
+-----------------------------------------------------+------+----------+
| MAYBE0-ADD:Check-Multiple-score,distance,name       |    7 |  0.64755 |
| Notfound1:has name - please debug                   |   87 |  8.04810 |
| OK-ADD:suggested for add-N1Full-name-match          |   10 |  0.92507 |
| OK-ADD:suggested for add-N2Label-name-match         |    3 |  0.27752 |
| OK-ADD:suggested for add-N3Unaccent-name-match      |  387 | 35.80019 |
| OK-ADD:suggested for add-N4Alias-name-match         |    3 |  0.27752 |
| OK-MultipleMathch-ADD                               |    1 |  0.09251 |
| SUGGESTION11-ADD:High-score,    check distance,name |   17 |  1.57262 |
| SUGGESTION12-ADD:Medium-score,  check distance,name |   97 |  8.97317 |
| SUGGESTION13-ADD:Low-score,     check distance,name |  469 | 43.38575 |
| -- total --                                         | 1081 |          |
+-----------------------------------------------------+------+----------+
(11 rows)
```

**Comment by nvkelso** [comment]
Huh, I'll fix that in v5.0!

**Comment by nvkelso** [comment]
Fixed via https://github.com/nvkelso/natural-earth-vector/pull/534.
